onnx/defs/experiments/defs.cc

```diff
@@ -364,7 +364,7 @@ OPERATOR_SCHEMA(Embedding)
         {"tensor(float16)", "tensor(float)", "tensor(double)"},
         "Constrain output types to float tensors.");
 
-OPERATOR_SCHEMA(ResizeNearest)
+OPERATOR_SCHEMA(Upsample)
```

Comment:
Add in the description that height_scale and width_scale need to be greater than 1.0.